amd_openvx_extensions/amd_rpp/source/ColorTwistbatchPD.cpp

```diff
@@ -190,50 +203,52 @@ static vx_status VX_CALLBACK uninitializeColorTwistbatchPD(vx_node node, const v
 	free(data->srcBatch_height);
-	delete(data);
-	return VX_SUCCESS; 
+	delete (data);
+	return VX_SUCCESS;
 }
 
 vx_status ColorTwistbatchPD_Register(vx_context context)
 {
 	vx_status status = VX_SUCCESS;
 	// Add kernel to the context with callbacks
 	vx_kernel kernel = vxAddUserKernel(context, "org.rpp.ColorTwistbatchPD",
-		VX_KERNEL_RPP_COLORTWISTBATCHPD,
-		processColorTwistbatchPD,
-		10,
-		validateColorTwistbatchPD,
-		initializeColorTwistbatchPD,
-		uninitializeColorTwistbatchPD);
+									   VX_KERNEL_RPP_COLORTWISTBATCHPD,
+									   processColorTwistbatchPD,
+									   10,
+									   validateColorTwistbatchPD,
+									   initializeColorTwistbatchPD,
+									   uninitializeColorTwistbatchPD);
 	ERROR_CHECK_OBJECT(kernel);
 	AgoTargetAffinityInfo affinity;
-	vxQueryContext(context, VX_CONTEXT_ATTRIBUTE_AMD_AFFINITY,&affinity, sizeof(affinity));
+	vxQueryContext(context, VX_CONTEXT_ATTRIBUTE_AMD_AFFINITY, &affinity, sizeof(affinity));
 #if ENABLE_OPENCL
 	// enable OpenCL buffer access since the kernel_f callback uses OpenCL buffers instead of host accessible buffers
 	vx_bool enableBufferAccess = vx_true_e;
-	if(affinity.device_type == AGO_TARGET_AFFINITY_GPU)
+	if (affinity.device_type == AGO_TARGET_AFFINITY_GPU)
 		STATUS_ERROR_CHECK(vxSetKernelAttribute(kernel, VX_KERNEL_ATTRIBUTE_AMD_OPENCL_BUFFER_ACCESS_ENABLE, &enableBufferAccess, sizeof(enableBufferAccess)));
 #else
 	vx_bool enableBufferAccess = vx_false_e;
 #endif
 	if (kernel)
 	{
-        unsigned short idx = 0; // For Index
+		unsigned short idx = 0; // For Index
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_IMAGE, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_OUTPUT, VX_TYPE_IMAGE, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
-        PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
+		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_ARRAY, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_SCALAR, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxAddParameterToKernel(kernel, idx++, VX_INPUT, VX_TYPE_SCALAR, VX_PARAMETER_STATE_REQUIRED));
 		PARAM_ERROR_CHECK(vxFinalizeKernel(kernel));
 	}
 	if (status != VX_SUCCESS)
 	{
-	exit:	vxRemoveKernel(kernel);	return VX_FAILURE; 
- 	}
+	exit:
```

Comment:
Codacy found an issue: [Label 'exit' is not used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)